openapi/components/schemas/Websites/Website.yaml

```
@@ -35,10 +35,8 @@ properties:
     $ref: ./WebsiteSettings.yaml
   organizationId:
     deprecated: true
-    readOnly: true
-    description: Organization ID.
     allOf:
-      - $ref: ../ResourceId.yaml
+      - $ref: ../OrganizationId.yaml
   _links:
     type: array
     description: The links related to resource.
```

nitpick: Where possible, we should avoid starting descriptions with "the"

This can be tiresome to read

```suggestion
    description: Related resource links.
```